aws_lambda_builders/workflows/ruby_bundler/bundler.py

```diff
@@ -66,6 +66,12 @@ def run(self, args, cwd=None):
                     self.osutils.remove_directory(check_dir)
             else:
                 # Bundler has relevant information in stdout, not stderr.
-                raise BundlerExecutionError(message=out.decode("utf8").strip())
+                if out and err:
+                    message_out = out.strip() + b"\n" + err
+                elif out:
+                    message_out = out
+                else:
+                    message_out = err
+                raise BundlerExecutionError(message=message_out.decode("utf8").strip())
```

Comment:
We might consider logging both stdout and stderr at debug level. So that if there is more information customers can check the logs by running `sam build --debug`.